### PR TITLE
H3nx/credits

### PR DESCRIPTION
**Pull request by h3nx**

### CODE REVIEW

File: SPLASH/src/game/states/MenuState.cpp [hunk 1]
```diff
@@ -382,6 +385,36 @@ void MenuState::renderMenu() {
...
+					ImGui::PopStyleColor();
+				}
+			}
+			if (m_windowToRender == 5) {
+				ImGui::PopStyleColor();
+			}
+
+
+
+		}
```
Comment (Smaugmuel) [inline SPLASH/src/game/states/MenuState.cpp]: Probably doesn't need to change, but why the extra scope?

File: SPLASH/src/game/states/MenuState.cpp [hunk 2]
```diff
@@ -716,6 +749,94 @@ void MenuState::renderOptions() {
...
+	names["Wester"] = { "Alexander", "person of pew" };
+	names["Bj�rk"] = { "Gustav", "person of scuffed newton" };
+	names["Wahl"] = { "Emil", "person of slave driving" };
+	names["Fredriksson"] = { "Daniel", "person of Github love" };
+	names["Glandberger"] = { "Oliver", "person of curry (with salad)" };
+	names["Fasth"] = { "Tobias", "person of wet balls" };
+	names["Enfeldt"] = { "Viktor", "person of Synchronization" };
+	names["Asp"] = { "Samuel", "person of pain and suffering" };
+	names["Meunier"] = { "Peter", "person of clutter" };
+	names["Bengtsson"] = {"David", "person of Jet engines" };
```
Comment (Smaugmuel) [inline SPLASH/src/game/states/MenuState.cpp]: Does not really matter, but be aware that this code will make a static map only once, but still update the names every function call.

File: SPLASH/src/game/states/MenuState.cpp [hunk 3]
```diff
@@ -716,6 +749,94 @@ void MenuState::renderOptions() {
 		m_optionsWindow.renderWindow();
 	}
 	ImGui::End();
+}
+void MenuState::renderCredits(float dt) {
```
Comment (Smaugmuel) [inline SPLASH/src/game/states/MenuState.cpp]: No need to change, but delta time is not used in this function

File: SPLASH/src/game/states/MenuState.cpp (hunk 2 above)
Comment (h3nx) [reply SPLASH/src/game/states/MenuState.cpp]: oh right, didnt think about that

**Pull request merged**